modules/linear_boltzmann_solvers/lbs_problem/acceleration/nl_keigen_acc_residual_func.cc

```diff
@@ -18,7 +18,7 @@ NLKEigenAccResidualFunction(SNES snes, Vec phi, Vec r, void* ctx)
 
   auto& diff_solver = nl_context_ptr->diff_solver;
 
-  auto& lbs_problem = nl_context_ptr->lbs_problem;
+  auto& lbs_problem = nl_context_ptr->do_problem;
```

Comment:
Why is the result still called `lbs_problem`?

Reply:
@ragusa, @wdhawkins  When fixing this, I noticed that there are still a few `lbs_problem` user params. I renamed the `do_problem` to `problem`, so should I also rename `lbs_problem` input parameter to `problem`?